lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3977,17 +3987,24 @@ template <typename AtenOpT, typename tosaOp>
 static Type getOutputTypeForNonAdaptivePoolingOp(
     RankedTensorType inputTy, SmallVectorImpl<int64_t> &kernelSize,
     SmallVectorImpl<int64_t> &strideArray, SmallVectorImpl<int64_t> &padArray,
-    SmallVectorImpl<int64_t> &dilationArray) {
+    SmallVectorImpl<int64_t> &dilationArray,
+    bool ceilMode=false) {
   auto inputShape = makeShapeTorchCompatible(inputTy.getShape());
   auto inputRank = inputTy.getRank();
   auto inputElemTy = inputTy.getElementType();
 
   int64_t outputHDim = ConvertAtenPoolingBaseOp<AtenOpT, tosaOp>::getOutputDim(
       inputShape[inputRank - 2], kernelSize[0], strideArray[0], padArray[0],
-      padArray[0], dilationArray[0]);
+      padArray[0], dilationArray[0], ceilMode);
   int64_t outputWDim = ConvertAtenPoolingBaseOp<AtenOpT, tosaOp>::getOutputDim(
       inputShape[inputRank - 1], kernelSize[1], strideArray[1], padArray[1],
-      padArray[1], dilationArray[1]);
+      padArray[1], dilationArray[1], ceilMode);
+  padArray[0] = (outputHDim - 1) * strideArray[0] + 
```

Comment:
Code formatting looks a bit inconsistent here (sometimes a*b and others a * b). Can you verify you've run clang-format? `git clang-format HEAD~1` should only modify lines you have already changed.
https://github.com/llvm/torch-mlir/wiki/Coding-Style

Reply:
Thanks Eric for your reviews. I've addressed your comments.